lib/rules.js

```diff
@@ -258,8 +258,14 @@ GnipRules.prototype.update = function (rules, cb) {
           fs.writeFile(self._cacheFile, self.live.parser.stringify(rules), 'utf8', cb);
         }
       ], function (err) {
-        if (err) cb(err);
-        else {
+        if (err) {
+          cb(err, {
```

Comment:
I don't think this is the best way to handle the error. Why not just throw an error with a proper message in `_deleteRulesBatch` and `_addRulesBatch`?
Also, you are passing both en error instance and an object as the 'result' value. This is not consistent, as the callback should expect only one of those.

Reply:
I totally agree, @demian85. I'll change the error message and update de PR.

Thanks!

Reply:
PR updated!
Best,
Enrique